opencensus/trace/ext/grpc/client_interceptor.py

```diff
@@ -86,6 +87,16 @@ def _start_client_span(self, client_call_details):
             attribute_key=attributes_helper.GRPC_ATTRIBUTES.get(GRPC_METHOD),
             attribute_value=str(client_call_details.method))
 
+        # Add user defined auxiliary attributes to span
+        if type(self.aux_attributes) is dict:
+            for attr_key, attr_value in self.aux_attributes.items():
+                if type(attr_key) is not str and type(attr_value) is not str:
+                    continue
```

Comment:
It looks like you're missing some coverage here: `...98%   94, 91->100, 93->94`.

You can run `coverage report -m -i $(git diff --name-only master)` after running the tests with `--cov=opencensus` to check uncovered lines.